scripts/basic-workflow-tests.sh

```diff
@@ -88,14 +93,9 @@ set_defaults() {
     NUM_INSTRUCTIONS=1
     TRAIN_ARGS+=("--num-epochs" "1")
 
-    if [ "${GRANITE}" -eq 1 ] && [ "${MIXTRAL}" -eq 1 ]; then
-        echo "ERROR: Can not specify -g and -M at the same time."
-        exit 1
-    fi
-
-    if [ "${MIXTRAL}" -eq 1 ] && [ "${BACKEND}" = "vllm" ]; then
-        echo "ERROR: Can not specify -M and -v at the same time."
-        exit 1
+    if [ "${EVAL}" -eq 1 ] && [ "${BACKEND}" != "vllm" ]; then
+         echo "ERROR: Can must specify -e and -v at the same time."
```

Comment:
The message is quite confusing. Should it be: `ERROR: can not use eval if the backend is not vLLM`? Or did you mean `ERROR: must specify -e and -v at the same time.`?

Reply:
Good catch, your suggestion for the error message is what I meant.